packages/manager/src/features/CloudPulse/shared/CloudPulseResourcesSelect.tsx

```diff
@@ -100,6 +119,25 @@ export const CloudPulseResourcesSelect = React.memo(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [resources, region, xFilter, resourceType]);
 
+    // selected resources will appear at the top in the autcomplete popper
+    const resourcesWithSelectedFirst = React.useMemo<
```

Comment:
rearranging options again & again will make Autocomplete to recreate options on every selection. Anyways autocomplete is adding selected options on the input field, rearranging it will not add any value. Any thoughts @kmuddapo  @venkymano-akamai